docs/get-started/elements.md

```diff
@@ -4,24 +4,25 @@ title: "Template and Slot Elements"
 
 # Template and Slot Elements
 
-The Web Components specification brings two elements to the Web Platform, `<template>` and `<slot>`, for creating content fragments and passing content in to components.
+There are two elements which can be very useful in creating Web Components which you might not be familliar with: `<template>` and `<slot>`, for creating content fragments and passing content in to components.
```

Comment:
Your correction didn't read well and had a typo. Something like this?

Two handy elements for creating content fragments and passing content to components you might not be familiar with are `<template>` and `<slot>`.